### DISCUSSION TITLE
Two Chairs Therapy. Engage with your-virtual-self.【Chen Xiangquan】

### DISCUSSION THREAD

**Discussion by ChenXiangquan915** [realitydeslab/playshop-summer-2024#27] [Proposal]
Research Questions: 研究问题:

How can engaging in self-dialogue within virtual reality aid in psychological healing by exploring different aspects of our emotions and personality?
通过在虚拟现实中进行自我对话，如何帮助我们探索自己情感和个性的不同方面，从而实现心理疗愈？

Keywords: 关键词:
AI, Psychological Health, Virtual Reality, Digital Identity 
人工智能，心理健康，虚拟现实，数字身份

Artefact / Media: 文物/媒体:

vr portrait collection in virtual reality
虚拟现实中的vr人像集

Abstract: 摘要:

This is a method to achieve self-healing in the psyche through dialogue with the past self in virtual reality, utilizing Gestalt therapy. This method can help us explore different aspects of our emotions and personalities, and through dialogue, it can help us understand and integrate emotions, thoughts, and opinions, thereby achieving emotional resolution and a stronger inner world. This is a form of psychological treatment that emphasizes personal responsibility, focusing on a person's present-moment experience, the environment and social background of one's life, as well as the self-adjustment and adaptation made by people due to the overall situation.

通过在虚拟现实中与过去的自己进行对话沟通，利用格式塔疗法实现自我心理疗愈的过程。这种方法可以帮助我们探索自己的情感和个性不同方面，通过对话帮助我们理解和整合情绪、想法和观点，从而实现情绪解决和更加强大的内心世界。这是一种心理治疗形式，它强调个人责任，关注个人在当下的体验、一个人生活的环境和社会背景，以及人们因整体情况而做出的自我调节调整。

The idea is presented as an extended folder in a VR world, where at each stage of time, you can upload information about yourself and create a virtual avatar similar to you. You can choose to communicate with yourself at different life stages: one is the present you, and the other is another part of you. Through face-to-face Q&A, you can achieve psychological healing. This folder can record your changes and also allow you to improve yourself through dialogue at the same time.

我的想法会呈现为在VR世界里一个延伸出来的夹，每一个时间阶段你都可以将自己进行信息上传，建造一个与你相似的虚拟形象，你可以选择你不同人生阶段的你进行对话沟通，一个是现在的你，另一个则是你的另一部分。通过你们面对面的问答来得到心理疗愈，这个夹可以记录你的变化，也可以让同一时间的你通过对话自我提升。

Your Reading List: 
您的阅读清单:

Visual Reference / Illustrations:
视觉参考/插图:

![屏幕截图 2024-06-28 142201](https://github.com/realitydeslab/playshop-summer-2024/assets/173972702/4e88ca89-dbe1-4e05-be02-a871abb85bdc)

References: 引用：
[1]ACM Conferences. 《Whole-Body Imitation of Human Motions with a Nao Humanoid | Proceedings of the Seventh Annual ACM/IEEE International Conference on Human-Robot Interaction》. World. 见于 2024年6月27日. https://doi.org/10.1145/2157689.2157830.
[2] ACM Other conferences. 《Active Ageing with Avatars | Proceedings of the 28th Australian Conference on Computer-Human Interaction》. World. 见于 2024年6月27日. https://doi.org/10.1145/3010915.3010944.
[3]ACM Other conferences. 《Technology and Aging | Proceedings of the 4th EAI International Conference on Smart Objects and Technologies for Social Good》. World. 见于 2024年6月27日. https://doi.org/10.1145/3284869.3284884.
[4]《AI行为科学：大语言模型如何模仿人类行为特征-北京大学光华管理学院》. 见于 2024年6月27日. https://www.gsm.pku.edu.cn/info/1316/29128.htm.
[5]3D建模学习工作室. 《使用生成式 AI 模仿人类行为》, 2023年8月8日. https://www.mvrlink.com/using-generative-ai-to-mimic-human-behavior/.
[6]Jones, Andrew, Jonas Unger, Koki Nagano, Jay Busch, Xueming Yu, Hsuan-Yueh Peng, Oleg Alexander, Mark Bolas和Paul Debevec. 《An automultiscopic projector array for interactive digital humans》. 收入 ACM SIGGRAPH 2015 Emerging Technologies, 1. SIGGRAPH ’15. New York, NY, USA: Association for Computing Machinery, 2015. https://doi.org/10.1145/2782782.2792494.
[7]Shen, Tong, Jiawei Zuo, Fan Shi, Jin Zhang, Liqin Jiang, Meng Chen, Zhengchen Zhang, Wei Zhang, Xiaodong He和Tao Mei. 《ViDA-MAN: Visual Dialog with Digital Humans》. 收入 Proceedings of the 29th ACM International Conference on Multimedia, 2789–91. MM ’21. New York, NY, USA: Association for Computing Machinery, 2021. https://doi.org/10.1145/3474085.3478560.
[8]Verywell Mind. 《How Gestalt Therapy Works》. 见于 2024年6月29日. https://www.verywellmind.com/what-is-gestalt-therapy-4584583.

**Comment by glitch-kraft** [comment]
Suggested title
Tracking Brain Age Performance

**Comment by glitch-kraft** [comment]
Look into EEG Technologies to study brain
https://openbci.com/

**Comment by glitch-kraft** [comment]
two chairs technique in psychology
Bring this to VR
https://haven.ca/6-steps-2-chair-process-gestalt-therapy-home/

**Reply by glitch-kraft** [reply]
As a from of XR psychotherapy based on your data portrait (across various ages)

**Reply by glitch-kraft** [reply]
![1d130814f8057cdea6ace419947161d5](https://github.com/realitydeslab/playshop-summer-2024/assets/46427283/096d69cd-73bf-4a5e-a9f8-6abd5ac6ac5c)

Back up of your data portrait for two chair technique (psychology)

**Comment by botaohu** [comment]
https://www.rewind.ai/

**Comment by botaohu** [comment]
Yes, the therapy you're referring to is known as "Gestalt therapy", specifically a technique called the "empty chair" or "two-chair" technique. This method is commonly used to help individuals explore different aspects of their personality, emotions, and experiences.

How It Works
Two Chairs: In this technique, two chairs are set up facing each other. One chair represents your current self or a particular aspect of yourself, while the other chair represents another part of yourself, a significant person in your life, or an issue you're dealing with.
Dialogue: You move between the two chairs, speaking from the perspective of the different roles or aspects. For example, you might sit in one chair and express your current feelings or thoughts, then move to the other chair and respond from the perspective of the other part of yourself or the significant person.
Integration: The goal is to create a dialogue that helps you understand and integrate conflicting emotions, thoughts, or perspectives. This process can lead to greater self-awareness and emotional resolution.

**Comment by ChenXiangquan915** [comment]
<img width="1440" alt="Desktop - 1" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173972702/0e4dcad1-c2c4-464c-ba3b-6d90acc2951c">
Tagline：How can engaging in self-dialogue within virtual reality aid in psychological healing by exploring different aspects of our emotions and personality?
Video prompts："Move the mouse to slide the central scroll button, and you will enter the time machine. On the right side, there are two buttons to adjust the time. As you adjust the page, you can choose to converse with different stages of yourself."

**Comment by glitch-kraft** [comment]
5 y.o.
10  y.o
10s
20s
30s
...

**Comment by glitch-kraft** [comment]
**Title**
Two Chairs Therapy. Engage with your-virtual-self. 

**Tagline**
Two Chairs - Generative Mixed Reality Therapy tailored to you. Talk to any age version of yourself based on your accurate and encrypted data portrait.

**Comment by ChenXiangquan915** [comment]
artifact:
https://www.figma.com/proto/ubqCWQ2xxZ9wcJQSmEorPH/Untitled?page-id=0%3A1&node-id=65-519&viewport=1042%2C691%2C0.23&t=N0ChsLMvKlgGnI4V-1&scaling=min-zoom&content-scaling=fixed
Opening may require agreeing to the invitation.

**Comment by ChenXiangquan915** [comment]
[Uploading Two Chairs Therapy. Engage with your-virtual-self.【Chen Xiangquan】.pdf…]()
